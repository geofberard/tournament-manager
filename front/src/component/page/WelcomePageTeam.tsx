import { Grid, SxProps } from "@mui/material";
import * as React from "react";
import { FC } from "react";
import { CompetitionBoard } from "../view/CompetitionBoard";
import { useCurrentTeam } from "../hook/CurrentTeamContext";
import { GeneralMenu } from "../view/GeneralMenu";
import { GameList } from "../view/GameList";

const fullScreen: SxProps = {
    height: "100vh",
};

export const WelcomePageTeam: FC = () => {
    const [currentTeam,] = useCurrentTeam();

    return (
        <>
            <Grid container spacing={0} direction={{ md: "row" }} sx={fullScreen}>
                <GeneralMenu />
                <CompetitionBoard />                
                <GameList />
            </Grid>
        </>
    );
};
