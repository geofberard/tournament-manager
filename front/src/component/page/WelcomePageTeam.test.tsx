import * as React from "react";
import { testSnapshot } from "../../test/TestUtils";
import { WelcomePageTeam } from "./WelcomePageTeam";

testSnapshot(<WelcomePageTeam />);
