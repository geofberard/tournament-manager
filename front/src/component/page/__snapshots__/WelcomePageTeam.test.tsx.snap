// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match last snapshot 1`] = `
<React.Fragment>
  <ForwardRef(Grid)
    container={true}
    spacing={0}
    sx={
      Object {
        "height": "100vh",
      }
    }
  >
    <GeneralMenu />
    <CompetitionBoard />
    <GameList
      games={
        Array [
          Game {
            "court": "court",
            "getResultGameByTeam": [Function],
            "id": "gameId",
            "isFinished": [Function],
            "referee": Object {
              "id": "teamC",
              "name": "TeamC",
            },
            "scoreA": 25,
            "scoreB": 14,
            "teamA": Object {
              "id": "teamA",
              "name": "TeamA",
            },
            "teamB": Object {
              "id": "teamB",
              "name": "TeamB",
            },
            "time": 2022-08-19T13:00:00.000Z,
          },
          Game {
            "court": "court",
            "getResultGameByTeam": [Function],
            "id": "gameId",
            "isFinished": [Function],
            "referee": Object {
              "id": "teamC",
              "name": "TeamC",
            },
            "scoreA": 14,
            "scoreB": 25,
            "teamA": Object {
              "id": "teamA",
              "name": "TeamA",
            },
            "teamB": Object {
              "id": "teamB",
              "name": "TeamB",
            },
            "time": 2022-08-19T14:00:00.000Z,
          },
          Game {
            "court": "court",
            "getResultGameByTeam": [Function],
            "id": "gameId",
            "isFinished": [Function],
            "referee": Object {
              "id": "teamC",
              "name": "TeamC",
            },
            "scoreA": 22,
            "scoreB": 22,
            "teamA": Object {
              "id": "teamA",
              "name": "TeamA",
            },
            "teamB": Object {
              "id": "teamB",
              "name": "TeamB",
            },
            "time": 2022-08-19T15:00:00.000Z,
          },
        ]
      }
      title="Vos Matchs"
    />
  </ForwardRef(Grid)>
</React.Fragment>
`;
